src/main/java/dev/emi/trinkets/mixin/InventoryScreenMixin.java

```diff
@@ -157,23 +189,64 @@ private void drawForeground(MatrixStack matrices, int mouseX, int mouseY, Callba
 	@Unique
 	private void drawGroup(MatrixStack matrices, SlotGroup group) {
 		RenderSystem.enableDepthTest();
-		this.setZOffset(310);
+		this.setZOffset(305);
```

Comment:
I'm curious what this change is for

Reply:
It's just to fix this rendering issue with the slot borders overlapping the slot backgrounds:

![example](https://user-images.githubusercontent.com/29991504/118864097-7f93df00-b894-11eb-9333-e0dbd94621da.png)

Admittedly I'm not super familiar with the intricacies of the z-offsets so I'm not sure if this has any unwanted side effects, or if there's a better number for it as I chose 305 pretty arbitrarily.